src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -60,6 +60,13 @@ private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate
         }
 
         [Fact]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.Net45 |
+            TargetFrameworkMonikers.Net451 |
+            TargetFrameworkMonikers.Net452 |
+            TargetFrameworkMonikers.Net46 |
```

Comment:
Crypto support started with netstandard1.3 (aligned with net46).  TargetFrameworkMonikers needed to go back to netstandard1.0 (net45).  But netstandard2.0 is (sort of) net461, so we shouldn't need to talk about any version older than that except for in libraries explicitly doing work further back.